tests/test_approval_modules_snapshots/cdf_infield_location.yaml

```diff
@@ -7,47 +7,6 @@ DataSet:
   externalId: ds_infield_default_location_source
   name: Infield default location source data set
 Group:
-- capabilities:
```

Comment:
I agree, it looks like the test before was wrong and that module `cdf_infield_location` should not generate any capability that is `all` scoped.